src/utils/sanitizers.ts

```diff
@@ -0,0 +1,4 @@
+export const dropLeadingInfo = (str: string) => {
+  // remove everything before (and including) the first space
+  return str.replace(/^\S+\s+/g, '');
```

Comment:
Also considered matching just the first few occurrences of the characters we don't want, but it ended up being a bit more complex to limit the matching to only the first ~5 characters. So I ended up going with a simpler approach to just drop everything before and with the first space.

The alternative (for reference)
```
  return str.replace(/[^a-zA-Z]{2,4}/g, '');
```